src/pages/InviteReportParticipantsPage.tsx

```diff
@@ -161,6 +166,7 @@ function InviteReportParticipantsPage({betas, personalDetails, report, didScreen
         });
         Report.inviteToGroupChat(reportID, invitedEmailsToAccountIDs);
         Navigation.navigate(backRoute);
+        setSelectedOptions([]);
```

Comment:
Is this fixing some bug?

Reply:
It might not be needed. When testing I saw selected options persisting after inviting users and then returning to the invite page. But looking again and that's not happening anymore.